nailgun/entities.py

```diff
@@ -7580,6 +7580,72 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
         return super(System, self).read(entity, attrs, ignore, params)
 
 
+class TailoringFile(
+        Entity,
+        EntityCreateMixin,
+        EntityDeleteMixin,
+        EntityReadMixin,
+        EntitySearchMixin,
+        EntityUpdateMixin):
+    """A representation of a Tailoring File entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'name': entity_fields.StringField(
+                required=True,
+                str_type='alpha',
+                unique=True
+            ),
+            'scap_file': entity_fields.StringField(),
+            'original_filename': entity_fields.StringField(),
+            'tailoring_file_profiles': entity_fields.StringField(),
+            'location': entity_fields.OneToManyField(Location),
+            'organization': entity_fields.OneToManyField(Organization),
+        }
+        if 'scap_file' in kwargs:
+            with open(kwargs['scap_file']) as input_file:
+                kwargs['scap_file'] = input_file.read()
+        self._meta = {
+            'api_path': 'api/v2/compliance/tailoring_files',
+            'server_modes': ('sat')
+        }
+        super(TailoringFile, self).__init__(server_config, **kwargs)
+
+    def create(self, create_missing=None):
+        """Fetch a complete set of attributes like locations, organizations for this entity.
```

Comment:
The override of function `create` is doing something else that just docstringed `here`. You can refer the mentioned bugzilla in that docstring.

Anyways its a non blocking comment,

Reply:
I was facing issue related to not getting full list of attributes and saw most of other entities were facing the same. I have followed same approach as them and they all were referring to same bz.